app/api/models/statistics.py

```diff
@@ -140,3 +141,36 @@ def __init__(self, **data):
         data["regulation_stats"] = data["_compiled_data"]
 
         super().__init__(**data)
+
+class ExampleObject(BaseModel):
+    type: str
+    id: str
+
+class ExampleObjects(BaseModel):
```

Comment:
It's recommended to use singular format for Model class names and in cases like this using something ExampleObjectList . 
At the first glance I didn't even notice plural s